recipes/seq-seq-pan/meta.yaml

```diff
@@ -24,6 +24,7 @@ requirements:
     - openjdk
     - argparse
     - mauvealigner ==1.2.0
+    - libgenome=1.3.1=h470a237_0
```

Comment:
What's wrong with the most recent build? This one only exists for Linux.

Reply:
It breaks progressiveMauve and I can't figure out why... I tried to fix it but my gcc knowledge is not sufficient (#15740 ). In the meantime issues are reported on my code because of that problem... (https://gitlab.com/chrjan/seq-seq-pan/issues/3 )

Reply:
Ah, yeah, that'll take quite some effort to figure out.